build.gradle

```diff
@@ -37,7 +37,7 @@ String dockerPrefix = "smarterbalanced"
 
 allprojects {
     group = 'org.opentestsystem.rdw.reporting'
-    version = '1.0.0-SNAPSHOT'
+    version = '1.0.1-SNAPSHOT'
```

Comment:
personally i'd change this to 1.1.0 but i realize you are just merging changes

Reply:
I meant to bring that up, our 'develop' versions seem to be a little behind.  I don't think this is the PR to fix it/them, but something to be aware of.